contracts/InterchainTokenService.sol

```diff
@@ -871,6 +845,16 @@ contract InterchainTokenService is
         if (!success) revert GatewayCallFailed(returnData);
     }
 
+    function callContractWithToken(
+        string calldata destinationChain,
+        string calldata contractAddress,
+        bytes calldata payload,
+        string calldata symbol,
+        uint256 amount
+    ) external override {
+        // do nothing ?
```

Comment:
I'm not sure what to do with the `callContractWithToken` function here because it's already implemented as `_callContractWithToken` and is called within `_transmitInterchainTransfer` and `interchainTransfer` (tightly coupled with the `metadata`). Could you give me some guidance?

Reply:
had a call and discussed.